app/containers/filterContainer/yearRangeSlider/index.tsx

```diff
@@ -192,6 +192,15 @@ const YearRangeSlider: React.FunctionComponent<YearRangeSliderProps> = props =>
                 fromBtn: true,
               });
             }}
+            onToggleClick={() => {
```

Comment:
I think it is too redundant and unnecessary.
Actually, you don't need this prop cuz you can make same logic inside of FilterButton component.

```js
onClick={() => {
  if (props.isActive) {
              goToYearFilteredSearchResultPage({
                qs: queryParamsStr,
                history: props.history,
                max: currentYear,
                min: MIN_YEAR,
                fromBtn: true,
              });
  } else {
    props.onClick();
  }
}}
```